__tests__/writePass.test.js

```diff
@@ -270,6 +270,8 @@ describe('Pass Successes', () => {
     const writePassHandler = require('../lambda/writePass/index');
     process.env.QR_CODE_ENABLED = "true";
     process.env.ADMIN_FRONTEND = "http://localhost:4300";
+    process.env.PASS_MANAGEMENT_ROUTE="/pass-management";
```

Comment:
Lint: Spaces around `=`

Reply:
Spaces added. I need to turn on my linter again. Sometimes it changes the entire file and I forgot to re-enable it.